frontend/ui/widget/infomessage.lua

```diff
@@ -53,6 +53,7 @@ local InfoMessage = InputContainer:extend{
     _timeout_func = nil,
     width = nil,  -- The width of the InfoMessage. Keep it nil to use default value.
     height = nil,  -- The height of the InfoMessage. If this field is set, a scrollbar may be shown.
+    cap_height_to_one_line = false,  -- Attempt to show text in one single line. This setting and height are not to be used conjointly.
```

Comment:
Feels this is still not a good name.
It explains the how we achieve one line (explanation which is not really needed) and moreover it's wrong (we don't cap the height - which could invoke crop & scroll -, we reduce the font size).
So, single_line or force_one_line.